### PR TITLE
Added smoke-test action

### PR DESCRIPTION
**Pull request by johnsca**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,193 +37,199 @@ unit that communicates with the cluster by relating to the
 
     juju-quickstart apache-hadoop-spark
```
Comment (kwmonroe) [inline README.md]: Let's change this to either:

``juju deploy apache-hadoop-spark``

or

``juju deploy bundle/apache-hadoop-spark``

File: actions/smoke-test [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# parameterized from this outer scope
+su ubuntu << EOF
+. /etc/environment
+echo 'calculating pi'
+spark-submit --master \${MASTER} --class org.apache.spark.examples.SparkPi \
+  /usr/lib/spark/lib/spark-examples-*.jar ${NUM_PARTITIONS} &> ${RESULT_LOG}
+EOF
+STOP=`date +%s`
+
+DURATION=`expr $STOP - $START`
```
Comment (kwmonroe) [inline actions/smoke-test]: do we need to action-set any output here?  like "pi is roughly 3.x" from the output of spark-submit, or is that somehow implicitly returned in the action fetch output?

Comment (johnsca) [inline actions/smoke-test]: Updated

**Comment by kwmonroe** [comment]
Nice, smoke-test failure output look good.  We're keeping the quickstart references until 2.x lands.

+1, LGTM

**Pull request merged**